pkg/util/events.go

```diff
@@ -0,0 +1,51 @@
+package util
```

Comment:
You need the license boilerplate at the beginning of new Go files https://github.com/aws-controllers-k8s/memorydb-controller/blob/main/cmd/controller/main.go#L1-L12

Reply:
Added.